README.md

```diff
@@ -46,7 +46,6 @@ jupyter lab --watch
 To install the backend, assuming you've followed the conda based setup [xkcd example](http://jupyterlab.readthedocs.io/en/stable/developer/xkcd_extension_tutorial.html):
 ```
 conda install pip
-pip install datadots-api==0.1.2
```

Comment:
This is the Dotmesh api isn’t it? No longer used I guess.

Reply:
The Python package has a dependency on it already (and a newer version), so there's no need to explicitly install old version. Not sure if it's used, but this line is definitely unnecessary.